web/src/utils/columns.ts

```diff
@@ -201,16 +195,14 @@ export const getCommonColumns = (t: TFunction, withConcatenatedFields = true): C
       id: ColumnsId.port,
       name: t('Ports'),
       isSelected: false,
-      filterType: FilterType.PORT,
       value: f => getSrcOrDstValue(f.fields.SrcPort, f.fields.DstPort),
       sort: (a, b, col) => comparePorts(col.value(a) as number, col.value(b) as number),
       width: 10
     },
     {
       id: ColumnsId.host,
-      name: t('Hosts'),
+      name: t('Node IPs'),
```

Comment:
In terms of consistency, either choose between 'Addresses' or 'IPs' everywhere

Reply:
Maybe, I didn't use "Node Addresses" to save some space, but yes we could change Addresses to IPs
btw I'm not sure about the plural form, maybe should stick to just IP here? (IPs isn't pretty)
cc @nalhadef any advice here?

Reply:
@jotak @jpinsonneau Definitely not "IPs." :-)

I hate to say it since you already eliminated it due to space concerns, but "Node Addresses" is best. It's specific in a way that IP addresses is not.

Reply:
or an alternative: we just write "Node" .. at the moment it's still an IP, but perhaps we will show the node name at some point (Julien was planning to add node names anyway, maybe we don't need two different columns)

Reply:
I'd vote for to change all Addresses to IP. For me IP is more understandable than just "Address", of course "IP Address" would be best, but in the interest of space, IP and Node IP probably signifies what column is meant for as opposed to "Address" .

Reply:
+1 I'll do that